src/main/java/Deadline.java

```diff
@@ -0,0 +1,62 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+/**
+ * Represents a task to be completed.
+ * A <code>Deadline</code> object contains description of a task e.g., <code>Assignment 1</code>, the deadline date
+ * e.g., <code>2020-02-01</code> and the deadline time e.g., <code>1000</code> of the task.
+ */
```

Comment:
Good attempt to document your code! However, try to keep the [line length within 120 characters](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html#layout). You can change some settings in Intellij to help you with this :)